### PR TITLE
cloth masks

### PR DESCRIPTION
**Pull request by paul-hammant**
Pro **cloth** masks section if a person has to go into the public setting (including mass transport). 

https://www.youtube.com/watch?v=_JH04M04eQQ made a very strong case, pulling together evidence presented previously elsewhere

### CODE REVIEW

File: _sections/act_and_prepare/en/02-do/16-wear-a-mask-when-in-public.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{:.do}
+### Wear a facemask when in public
+
+Studies have shown for Influenza that a mask may have a self-protection benefit. The viron that causes COVID-19 is roughly the same size as the Influenza viron at 125 nanometers and ithas the same mechanisms of entering the human body. If you absolutely have to venture into public where other people may be, buy a cloth mask or make your own as US hospital chains are beginning to do.
```
Comment (jaysonvirissimo) [inline _sections/act_and_prepare/en/02-do/16-wear-a-mask-when-in-public.md]: Could we put each sentence on its own line? That way changes to a single sentence don't cause the whole paragraph to light up red in GitHub's diff UI.

Comment (jaysonvirissimo) [inline _sections/act_and_prepare/en/02-do/16-wear-a-mask-when-in-public.md]: "Viron" is scientific jargon that many readers won't understand. Maybe we could use the word "virus" even though it isn't as correct. Alternatively, we could add a parenthetical, like so:

"The viron (complete virus particle) that causes COVID-19 is roughly the same size..."

Comment (jaysonvirissimo) [inline _sections/act_and_prepare/en/02-do/16-wear-a-mask-when-in-public.md]: "ithas" is missing whitespace

Comment (nicolevasilevsky) [reply _sections/act_and_prepare/en/02-do/16-wear-a-mask-when-in-public.md]: need space between it has

Comment (nicolevasilevsky) [reply _sections/act_and_prepare/en/02-do/16-wear-a-mask-when-in-public.md]: +1

(I like either suggestion)

Comment (nicolevasilevsky) [reply _sections/act_and_prepare/en/02-do/16-wear-a-mask-when-in-public.md]: Oops, I see @jaysonvirissimo mentioned ithas below

Comment (jaysonvirissimo) [reply _sections/act_and_prepare/en/02-do/16-wear-a-mask-when-in-public.md]: Thanks for fixing this @steesdale 👍

**Pull request merged**